server/world/sound/item.go

```diff
@@ -58,3 +58,7 @@ type GoatHorn struct {
 
 	sound
 }
+
+// UseFireCharge is a sound played when a player lights a block on fire with a fire charge, or when a dispenser or a
+// blaze shoots a fireball.
+type UseFireCharge struct{ sound }
```

Comment:
Because this sound is also played when a blaze shoots a fireball, I think it might be better to just name this something like `FireCharge`.